evap/staff/views.py

```diff
@@ -283,6 +283,21 @@ def semester_lottery(request, semester_id):
     template_data =dict(semester=semester, form=form, eligible=eligible, winners=winners)
     return render(request, "staff_semester_lottery.html", template_data)
 
+@staff_required
+def semester_todo(request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+
+    lecturers = []
```

Comment:
this needs to be a set, otherwise contributors might be added more than once (and then you have to use ```lecturers.add``` instead of ```lecturers.append```).

also, please replace ```lecturers``` by ```responsibles```